testing/gpg-remailer/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=gpg-remailer
+pkgver=3.02.01
+pkgrel=0
+pkgdesc="decrypt messages and re-encrypt them to a well-defined group"
+url="https://fbb-git.github.io/gpg-remailer/"
+arch="all"
+license="GPL3+"
+makedepends="icmake bash yodl bobcat bobcat-dev"
+subpackages="$pkgname-doc"
+source="gpg-remailer-$pkgver.tar.gz::https://github.com/fbb-git/gpg-remailer/archive/$pkgver.tar.gz
+	avoid-gnuisms.patch"
+builddir="$srcdir/gpg-remailer-$pkgver/gpg-remailer"
+options="!check"
```

Comment:
If upstream does not provide any tests and the project is an application (not a library), then it’s better to add at least very basic check that just calls the produced binary with `--version`. Just to check that the produced binary is not totally broken.